### ISSUE TITLE
Feature Request: 

### ISSUE THREAD

**Issue by 4lk** [Xyphyn/photon#421]
**Is your feature request related to a problem? Please describe.**
This is admittedly a very personal thing. I am subscribed to approximately 70 communities so far and I have difficulties in following them in a way that makes sense to me. I feel my feed is already so mixed up. 

Apart from that, there are some communities I want to follow but since there are posts all day long, they overflow my feed. Also, there are some topics that I like to follow but due to this overflow, I noticed I miss posts that interest me. So I have to go specifically to each community separately, even if there are 5 on a topic I want to keep an eye on daily. Which means I have to remember them by name and go through a long list to find them.

**Describe the solution you'd like**
I would love to have folders which I would populate with communities by topic. This way my feed would be coherent depending on what I want to see. 

**Describe alternatives you've considered**
Maybe have at least some categorization options like Topic 1, Topic 2, Topic 3? Something like Starred communities but not just one, perhaps a few more? 

No matter what, thank you for Photon, it is a great interface.

**Comment by sevonj** [comment]
Community groups could be a part of lemmy itself, too.

**Comment by Xyphyn** [comment]
I actually planned to add this and am still brainstorming ways I can implement this. Lemmy doesn't have easy grouping like this, so I'd have to do this:

- Allow users to create their own "location" with custom communities, like a multi-reddit.
- This would take forever to load though, probably encountering rate limits. I could find a way to do it though.
- I'd have to implement the sorting algorithms for each myself unless i just want sequential items.

I'm experimenting with this right now on a different branch, if something works i'll announce it.

**Comment by sevonj** [comment]
https://join-lemmy.org/news/2024-09-11_-_New_NLnet_funding_for_Lemmy
https://github.com/LemmyNet/lemmy/issues/818

I see that Lemmy multi-communities are planned now, so I think developing a custom frontend-only solution would be a waste.

**Comment by wolfgang101** [comment]
> Lemmy doesn't have easy grouping

Even though this feature would be so nice to have... I do agree with @sevonj 

If there is no easy way to fetch content from multiple communities with a single call then it might be better to wait for the [lemmy implementation](https://github.com/LemmyNet/lemmy/issues/818#issuecomment-2726807332).

**Comment by Xyphyn** [comment]
Yes, my philosophy for the design of Photon has changed. I believe this is a feature that should be handled server-side, and adding it client side would just bloat the app up. It would also be ridiculously inefficient, and I'd have to implement sorting myself.

Same reason I'll not add keyword filtering. That's something for the server-side to handle.
